draft-ietf-idr-link-bandwidth.xml

```diff
@@ -170,26 +196,34 @@
         egress processing to adj-RIB-out only or in ingress processing in
         which case link bandwidth community is present in Local-RIB.</t>
 
-        <t> Note: Implementation may provide configuration option (knob) to
-        allow sending of non-transitive link bandwidth communities to external
-        BGP sessions.</t>
+        <t>Note: Implementation may provide configuration option (knob) to
+        allow sending of non-transitive link bandwidth extended community
+        towards external BGP sessions.</t>
       </section>
 
       <section anchor="Receiver"
                title="Receiver (Receiving link bandwidth community)&nbsp;">
         <t>A BGP receiver MUST be able to process link bandwidth community of
-        both transitive or non-transitive type. The receiver SHOULD NOT flap
-        or treat the route as malformed based on the transitivity of the link
+        both transitive or non-transitive type. The receiver MUST NOT flap or
+        treat the route as malformed based on the transitivity of the link
         bandwidth community and/or BGP session type (internal vs.
         external).</t>
+
+        <t>Note: Implementation may provide configuration option (knob) to
+        accept non-transitive link bandwidth extended community from external
+        BGP sessions.</t>
       </section>
 
       <section anchor="Re-advertisement" title="Re-advertisement Procedures">
         <section anchor="NextHopSelf"
                  title="Re-advertisement with Next hop Self">
           <t>When a BGP speaker re-advertises a route with the Link Bandwidth
           Extended Community and sets the next hop to itself, it SHOULD follow
-          the same procedures as outlined in <xref target="Originator"/>.</t>
+          the same procedures as outlined in <xref target="Originator"/>. If
+          the export policy does not change the Link Bandwidth Extended
```

Comment:
keepint this in a separate <t></t> may be better. suggestion:

</t> In the absence of any import, export policies that alter the LBW, a LBW extended community received (if any) on the route is re-advertised as is, following regular BGP procedures. </t>

Reply:
Hi Kali, 
Thanks for reviewing the document. I have made this change. Please review.